.editorconfig

```diff
@@ -0,0 +1,13 @@
+# https://editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[Makefile]
+indent_style = tab
+indent_size = 4
```

Comment:
This is editor-visual only, and therefore the user's preference -- we shouldn't enforce a tab display size.

Reply:
Good point! Will fix thanks